scripts/templates/create-component/EmptyTypes.mustache

```diff
@@ -53,10 +76,16 @@ export interface I{{componentName}}ViewProps extends I{{componentName}}Props {
   onClick?: (ev?: React.MouseEvent<HTMLElement>) => void;
 }
 
+/**
+* {@docCategory {{componentName}}}
+*/
 export interface I{{componentName}}Tokens {
   // Define tokens for your component here. Tokens are styling 'knobs' that your component will automatically
   // apply to styling sections in the styles file.
   textColor?: string;
 }
 
+/**
+* {@docCategory {{componentName}}}
+*/
```

Comment:
I think this question came up before but is there no way to just specify this once per file?

Reply:
Not that I'm aware of. These tags are stored in the doc comment of each API in the api.json file that api-extractor generates. It has no knowledge of what API is coming from what file.

Reply:
Hmm would there be any way to validate that types files provide them as part of a linting step?

Reply:
Hmm there might be - I can look into it!